### PR TITLE
features: Expose idmap support

### PR DESCRIPTION
**Pull request by rata**
This PR just exposes the idmap support in the "features" subcommand.

cc @cyphar @AkihiroSuda

### CODE REVIEW

**Comment by rata** [comment]
Hmm the `features` subcommand seems to fail on rootless, but I can't repro locally

**Comment by rata** [comment]
Can someone add the changelog impact labels?

**Review by fuweid — APPROVED** [review]
LGTM(nb)

File: tests/integration/features.bats [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	# pointing to another user's directory after sudo rootless.
+	requires root
+}
+
+@test "features [idmap]" {
+	runc features
+	[ "$status" -eq 0 ]
+	enabled=$(echo "$output" | jq '.linux.mountExtensions.idmap.enabled')
+	[[ "$enabled" == "true" ]]
+}
```
Comment (AkihiroSuda) [inline tests/integration/features.bats:17]: This doesn't seem to worth testing

Comment (rata) [reply tests/integration/features.bats:17]: Oh, I was thinking to move this to unit tests. But sure, I'll remove it! :)

**Comment by rata** [comment]
@AkihiroSuda fixed, PTAL :)

**Review by AkihiroSuda — APPROVED** [review]

**Review by cyphar — APPROVED** [review]
lgtm

**Pull request merged**